gradle.properties

```diff
@@ -116,7 +116,7 @@ artifactRedirecting.androidx.annotation.version=1.9.1
 artifactRedirecting.androidx.graphics.version=1.0.1
```

Comment:
The code lgtm, but why CI fail? Does it fail in integration?

Reply:
yes